Cloudinary/Classes/Core/Features/Helpers/CLDExpression.swift

```diff
@@ -286,15 +286,16 @@ import Foundation
     // MARK: - provide content
     public func asString() -> String {
         
-        guard !currentKey.isEmpty && !currentValue.isEmpty else {
-            
-            return String()
-        }
+        guard !currentKey.isEmpty else { return String() }
         
-        let key   = replaceAllExpressionKeys(in: currentKey)
-        let value = removeExtraDashes(from: replaceAllUnencodeChars(in: currentValue))
+        let key = replaceAllExpressionKeys(in: currentKey)
         
-        return "\(key)_\(value)"
+        if currentValue.isEmpty {
+            return "\(key)"
+        } else {
```

Comment:
There is no need in `else` when there is a `return` statement in the positive flow.